tests/tests/dt-plugin-tests.cy.ts

```diff
@@ -253,38 +251,40 @@ describe('OpenShift Distributed Tracing UI Plugin tests', () => {
-    cy.get('.pf-v5-l-stack > .pf-v5-c-menu-toggle > .pf-v5-c-menu-toggle__text').click();
-    cy.get(':nth-child(1) > .pf-v5-c-menu__item > .pf-v5-c-menu__item-main > .pf-v5-c-menu__item-text').click();
-    cy.get('[style="height: auto; flex-direction: column-reverse;"] > .pf-v5-c-button').click();
+    cy.log('Assert traces in TempoStack instance.');
+    cy.get(':nth-child(1) > .pf-v6-c-toolbar__item > .pf-v6-c-form > .pf-v6-c-form__group > .pf-v6-c-form__group-control > .pf-v6-c-menu-toggle > .pf-v6-c-menu-toggle__button > .pf-v6-c-menu-toggle__controls > .pf-v6-c-menu-toggle__toggle-icon').click();
+    cy.get(':nth-child(2) > .pf-v6-c-menu__item > .pf-v6-c-menu__item-main > .pf-v6-c-menu__item-text').click();
+    cy.get(':nth-child(1) > :nth-child(2) > .pf-v6-c-form > .pf-v6-c-form__group > .pf-v6-c-form__group-control > .pf-v6-c-menu-toggle > .pf-v6-c-menu-toggle__button').click();
+    cy.get(':nth-child(2) > .pf-v6-c-menu__item > .pf-v6-c-menu__item-main > .pf-v6-c-menu__item-text').click();
+    cy.get(':nth-child(2) > .pf-v6-c-form__group > .pf-v6-c-form__group-control > .pf-v6-c-menu-toggle > .pf-v6-c-menu-toggle__controls > .pf-v6-c-menu-toggle__toggle-icon').click();
+    cy.get(':nth-child(1) > .pf-v6-c-menu__item > .pf-v6-c-menu__item-main > .pf-v6-c-menu__item-text').click();
+    cy.get(':nth-child(1) > :nth-child(2) > .pf-v6-c-form > .pf-v6-c-form__group > .pf-v6-c-form__group-control > .pf-v6-c-menu-toggle > .pf-v6-c-menu-toggle__button > .pf-v6-c-menu-toggle__controls > .pf-v6-c-menu-toggle__toggle-icon').click();
+    cy.get(':nth-child(1) > .pf-v6-c-menu__item > .pf-v6-c-menu__item-main > .pf-v6-c-menu__item-text').click();
+    cy.get('.pf-v6-c-toolbar__group > :nth-child(1) > .pf-v6-c-form > .pf-v6-c-form__group > .pf-v6-c-form__group-control > .pf-v6-c-menu-toggle > .pf-v6-c-menu-toggle__controls > .pf-v6-c-menu-toggle__toggle-icon').click();
+    cy.get(':nth-child(1) > .pf-v6-c-menu__item > .pf-v6-c-menu__item-main > .pf-v6-c-menu__item-text').click();
+    cy.get('.pf-m-toggle-group > .pf-v6-c-toolbar__group > :nth-child(2) > .pf-v6-c-form > .pf-v6-c-form__group > .pf-v6-c-form__group-control > .pf-v6-c-menu-toggle > .pf-v6-c-menu-toggle__button').click();
+    cy.contains('.pf-v6-c-menu__item-text', 'http')
+      .closest('.pf-v6-c-menu__item')
+      .find('input[type="checkbox"]')
+      .check();
+    cy.contains('.pf-v6-c-menu__item-text', 'grpc')
+      .closest('.pf-v6-c-menu__item')
+      .find('input[type="checkbox"]')
+      .check();
     cy.get('.MuiDataGrid-row--firstVisible > [data-field="name"] > .MuiBox-root > .MuiTypography-root').click();
     cy.get('[data-index="1"] > .css-1cqmfcw > .MuiStack-root > .MuiBox-root').click({ force: true });
-    cy.get('.css-tgncor').then(($el) => {
-      cy.log(`Actual text in .css-tgncor (TempoStack): ${$el.text()}`);
+    cy.get('.css-1bmckj4').then(($el) => {
```

Comment:
I think this class name is random and could change on every build.

Reply:
Ack, will change it.